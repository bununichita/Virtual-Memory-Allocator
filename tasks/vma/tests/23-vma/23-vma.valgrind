==20493== Memcheck, a memory error detector
==20493== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==20493== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==20493== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==20493== Parent PID: 20492
==20493== 
==20493== 
==20493== HEAP SUMMARY:
==20493==     in use at exit: 0 bytes in 0 blocks
==20493==   total heap usage: 12 allocs, 12 frees, 8,392 bytes allocated
==20493== 
==20493== All heap blocks were freed -- no leaks are possible
==20493== 
==20493== For lists of detected and suppressed errors, rerun with: -s
==20493== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
