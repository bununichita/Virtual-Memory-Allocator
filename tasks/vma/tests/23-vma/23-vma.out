GIGE
1
5
